src/pages/console/tutorial/buildui.mdx

```diff
@@ -3,7 +3,7 @@ export const meta = {
   description: `Figma to React code with Amplify Studio`
 };
 
-With Studio’s new “UI Library”, you can sync components from Figma to React code. Amplify also provides you with a handy Figma file to get started faster. The Amplify Figma file includes both UI primitives and component templates. You can also create your own component in Figma! In this tutorial, you'll create a home listings app using Amplify Studio.
+With Studio’s new **UI Library**, you can sync components from Figma to React code. Amplify also provides you with a handy Figma file to get started faster. The Amplify Figma file includes both UI primitives and component templates. You can also create your own component in Figma! In this tutorial, you'll create a home listings app using Amplify Studio.
```

Comment:
`handy Figma file` should be a link to the actual Figma community file.

Reply:
adding a link here is not necessary as we already provide the whole instructions in the section below.